File: Week 6/PSET 6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if h <= 8 and h >= 1:
        break
    else:
        continue
for i in range(h):
    print((h - 1 - i) * " " + (i + 1) * "#" + "  " + (i + 1) * "#")

File: Week 1/PSET 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int Hight;
    do
    {
        Hight = get_int("Hight:");
    }
    while (Hight < 1 || Hight > 8);

    for (int i = 0; i < Hight; i++)
    {
        for (int k = 0; k < Hight - (i + 1); k++)
        {
            printf(" ");
        }

        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}